modules/database/src/ioc/dbStatic/dbLexRoutines.c

```diff
@@ -1119,6 +1119,19 @@ static void dbRecordHead(char *recordType, char *name, int visible)
         return;
     }
 
+    if (recordType[0] == '#' && recordType[1] == 0) {
+        status = dbFindRecord(pdbentry, name);
+        if (status == 0) {
+            dbDeleteRecord(pdbentry);
+            fprintf(stderr, ERL_WARNING ": Record \"%s\" DELETED!\n", name);
+            return; /* done */
+        }
+        fprintf(stderr, ERL_ERROR ": Record \"%s\" not found\n", name);
+        yyerror(NULL);
```

Comment:
Should attempting to delete a non-existent record be an error?  Was this discussed already?

I do think that the warning on successful deletion should be removed.

Reply:
> Should attempting to delete a non-existent record be an error? Was this discussed already?

I did not get any feedback on that one so far. I guess it could be silently ignored and not raise an error. I copied this code from the `"*"` handling code.

> I do think that the warning on successful deletion should be removed.

Will remove.

Reply:
Addressed in [37c1e8b](https://github.com/epics-base/epics-base/pull/505/commits/37c1e8b78ef02954d835e0a3e0cf2c63989bf573)